build.gradle.kts

```diff
@@ -10,7 +10,14 @@ plugins {
 
 allprojects {
     group = "org.cru.godtools.kotlin"
-    version = "0.5.1-SNAPSHOT"
+
+    // configure the project version
+    if (!project.findProperty("releaseBuild")?.toString().toBoolean()) {
```

Comment:
I guess you can't go straight to boolean?

Reply:
it comes in as a `String`, but `findProperty()` returns it as an `Object` because `findProperty()` can actually return other types of objects as properties as well. Instead of casting it to `String`, I opted to use `.toString()` for a bit cleaner code.

In reality I'm not a fan of using `findProperty()` in general because it pulls from multiple datasources and isn't restricted to just properties passed from the command line. But I couldn't find a better method to use with a narrower scope.